### ISSUE TITLE
OPENCATS Email Configuration Testing

### ISSUE THREAD

**Issue by sidsingh1001** [opencats/OpenCATS#577]
#Not able to send email through SMTP
### Expected behavior and actual behavior.
I want to use SMTP for sending emails. However when i set Mailmailer to 3 in config file and test configuration it gives me error.
Have checked SMTP setting they are fine.
Copy them here just in case.
/* SMTP Settings. You don't need to worry about this unless MAIL_MAILER is
 * set to 3. If your server requires authentication, set MAIL_SMTP_AUTH to
 * true and configure MAIL_SMTP_USER and MAIL_SMTP_PASS.
 */
define('MAIL_SMTP_HOST', "smtppro.zoho.com");
define('MAIL_SMTP_PORT', "465");
define('MAIL_SMTP_AUTH', "true");
define('MAIL_SMTP_USER', "*******");
define('MAIL_SMTP_PASS', "******");
define('MAIL_SMTP_SECURE', "ssl");
//Options: '', 'ssl' or 'tls'
//define('MAIL_SMTP_SECURE', "tls");

### What version of opencats are you running? 
my OPenCats version is 0.9.6

#### Release or downloaded from Git? 
Release downloaded from GIT

#### WAMP or LAMP? 
Godaddy shared hosting LAMP

### What version of PHP and MySQL are you running
PHP version is 7.3.32
MySQL version is 5.6.43

**Comment by RussH** [comment]
Hi, 

please can you share the error message in the apache error log?

Also - PHP 7.2 is the maximum tested & supported. MariaDB v10.x (max 10.6) is required rather than MySQL as some defaults have diverged. 

Okay, so I just span up a test system and it emails fine - albeit using gmail rather than zoho. This is the gmail config that works;
define('MAIL_SMTP_HOST', "smtp.gmail.com");
define('MAIL_SMTP_PORT', 587);
define('MAIL_SMTP_AUTH', true);
define('MAIL_SMTP_USER', "username@domain");
define('MAIL_SMTP_PASS', "App_PASSWORD");
//Options: '', 'ssl' or 'tls'
define('MAIL_SMTP_SECURE', "tls");

**Comment by sidsingh1001** [comment]
Hi Russ

Thanks for the response Gmail works.
However the candidate once applied isnt able to login again, is there some configuration that i need to do?

Regards
Sid

**Comment by RussH** [comment]
no, this may be a 'feature' of using php 7.3 & MySQL unfortunately. I'd change  MySQL to MariaDB first and see. Also check your apache access log and error log when the candidate logs back in to see if there's anything complaining.

**Comment by sidsingh1001** [comment]
Thanks so much for the prompt response.
They are ale to apply so life isnt that bad :)
My hosting is shared hosting so dont have an option to move to MariaDB unfortunately.
But will keep this in mind for sure.

**Comment by RussH** [comment]
hi @sidsingh1001 it looks like the commit to fix this was just proposed, so it'll be tested  and then if it's working as proposed it'll be accepted .
